libs/artifacts/src/graphql/emissions/query.graphql

```diff
@@ -0,0 +1,87 @@
+fragment DialPreferences on Preference {
```

Comment:
It's possible we could have more fragments here, but it's not a huge set of queries so maybe not worth it